azurerm/internal/services/compute/ssh_public_key_resource.go

```diff
@@ -50,7 +50,7 @@ func resourceSshPublicKey() *schema.Resource {
 				),
 			},
 
-			"resource_group_name": azure.SchemaResourceGroupName(),
+			"resource_group_name": azure.SchemaResourceGroupNameDiffSuppress(),
```

Comment:
This is a bug on the API side - could we raise an issue on the SDK or api specs and link to it here?

Reply:
raised an issue in api specs

https://github.com/Azure/azure-rest-api-specs/issues/13491

Reply:
based on the response https://github.com/Azure/azure-rest-api-specs/issues/13491#issuecomment-801693754 , seems it's intended to be that way and not a bug.

Reply:
Thanks, regardless of that comment it is still IMHO a bug (even if they have made choices preventing them from fixing it) - according to the arm spec and reasonable expectations we should be getting the correct casing for the RG. As such could we please add a comment inline linking to that issue to document their response?

Reply:
@katbyte added inline comment. Thanks.